package.json

```diff
@@ -35,14 +35,15 @@
     "chai": "^3.5.0",
     "eslint": "^3.9.1",
     "eslint-config-screwdriver": "^2.0.9",
+    "hapi": "^16.1.0",
     "jenkins-mocha": "^4.1.1",
     "mockery": "^2.0.0",
-    "hapi": "^16.1.0",
     "sinon": "^1.17.7"
   },
   "dependencies": {
     "joi": "^10.2.2",
     "nodemailer": "^3.1.3",
+    "screwdriver-notifications-base": "^2.0.0",
```

Comment:
This is `3.0.0` now. 
I think previous builds for notifications-base had `BREAKING CHANGE` but never passed, and the newest build passed & took in account for all previous commit messages.